ssl/s3_lib.c

```diff
@@ -3729,6 +3729,14 @@ long ssl3_ctrl(SSL *s, int cmd, long larg, void *parg)
             return (int)s->ext.peer_ecpointformats_len;
         }
 
+    case SSL_CTRL_GET_IANA_GROUPS:
+        {
+            if (parg) {
```

Comment:
New code should arguably be checking `parg != NULL` rather than treating the pointer as a boolean.

Reply:
done.